main.tf

```diff
@@ -87,7 +87,7 @@ locals {
   }
 
   tags = {
-    dd_sls_terraform_module = "1.5.0"
+    dd_sls_terraform_module = "2.0.0"
```

Comment:
we're going to 2.0.0 because we're bumping the minimum required aws provider version.